### PR TITLE
YesbhautikX job list added to job-openings-data.json

### PR DESCRIPTION
**Pull request by yesbhautik**

### CODE REVIEW

File: public/data/job-openings-data.json [hunk 1]
```diff
@@ -231,6 +231,39 @@
...
+        },
+        {
+          "title": "Android Developer Intern [kotlin, java, flutter, other]",
+          "count": 4,
+          "experience": "1-2 Years as Fresher"
+        },
+        {
+          "title": "Sales Manager Intern",
+          "count": 2,
+          "experience": "1-2 Years as Fresher"
```
Comment (PareshMayani) [inline public/data/job-openings-data.json:259]: Position is for the Interns but You are looking for 1-2 years as Fresher? I mean Fresher and 1-2 years? Could you please clarify?

Comment (yesbhautik) [reply public/data/job-openings-data.json:259]: Like, worked on any serious personal projects, or done any internship.

Comment (PareshMayani) [reply public/data/job-openings-data.json:259]: @yesbhautik ok understood. Can you also please send some details about the company? Maybe the company registration certificate, company address, team size, etc.

**Comment by JaldeepAsodariya** [comment]
Closing this PR, We are inviting registered companies only.

**Comment by yesbhautik** [comment]
> Closing this PR, We are inviting registered companies only.

We are registered company only. Maintain some common sence how can I share all docs here publicly.

**Comment by PareshMayani** [comment]
@yesbhautik we can see common sense in the job description when someone is looking for 1-2 years experienced people as freshers. And another sense we can see in this statement "NodeJS & Python Developer Intern [Must have knowledge of both Language]". 

Let's not into that debate. When in the public forum, we should always be polite and respectful to each other. We are also volunteers contributing to the community. I hope you understand and respect it!

**Comment by yesbhautik** [comment]
> @yesbhautik we can see common sense in the job description when someone is looking for 1-2 years experienced people as freshers. And another sense we can see in this statement "NodeJS & Python Developer Intern [Must have knowledge of both Language]".
> 
> Let's not into that debate. When in the public forum, we should always be polite and respectful to each other. We are also volunteers contributing to the community. I hope you understand and respect it!

Now the topic is about registered company or not. It's basic stuff No one can share their company docs in public platforms. Anyone can misuse. Who will take responsibility of it after sharing document here! at public platforms.

**Comment by PareshMayani** [comment]
@yesbhautik Also, there were multiple reasons we found something wrong (one is listed below), but then we didn't have enough time to verify the details and documents because the event was almost near and there were a lot of tasks pending so we simply moved on.

- LinkedIn is showing 201-500 employees

**Comment by yesbhautik** [comment]
> @yesbhautik Also, there were multiple reasons we found something wrong (one is listed below), but then we didn't have enough time to verify the details and documents because the event was almost near and there were a lot of tasks pending so we simply moved on.
> 
> * LinkedIn is showing 201-500 employees

Again, its about registered company or not, nothing else as you mentioned when you was closing PR.

**Pull request closed without merging**